### DISCUSSION TITLE
HELP: Drag and Drop stopped Working

### DISCUSSION THREAD

**Discussion by Joshua82x** [tahoma2d/tahoma2d#1137] [Q&A]
Suddenly from one moment to another, drag and drop stopped working.
I could normally drag pictures from windows folders direect into my frames, step by step, suddenly it just dont allow me to drag anything into Tahoma2D anymore.

Does any why? 
or what has happened ?

**Comment by artisteacher** [comment]
I'm not sure what's going on, but have you tried restarting your computer?

**Comment by Joshua82x** [comment]
Tried restarting,.tried removing tahoma and deleting registry about it.
Tried running as admin, but it just shows a block sign when I draf any image from a folder into the program

________________________________

**Comment by artisteacher** [comment]
As long as it's a supported format, drag/drop should work in the Viewer and Timeline. Have you tried using the "Load Level" command instead?

**Comment by Joshua82x** [comment]
Checked everything it is a supported format, but still won't pull in.
Also I seem to have lost camera in the bottom line, or the other one, and can't find that section again.
Can add another camera it seems but not the other line thing.

________________________________

**Comment by manongjohn** [comment]
What type of file are you trying to drag in that suddenly stopped working?

What T2D version are you on?

If it's anything that goes through ffmpeg (mp3/mp4, jpg, etc...) it's possible something might have broken the connection between T2D and ffmpeg. Look at Preferences -> 3rd Party Appications.
